src/util.cpp

```diff
@@ -238,6 +241,18 @@ static void DebugPrintInit()
     mutexDebugLog = new boost::mutex();
 }
 
+static void DebugPrintInit1()
+{
+    assert(fileout1 == NULL);
+    assert(mutexDebugLog1 == NULL);
+
+    boost::filesystem::path pathDebug1 = GetDataDir() / "debug1.log";
```

Comment:
What is gonna be logged in debug.log and what to debug1.log?

Reply:
debug.log as always, debug1.log just cumulative emission data.